app/constants/colors.js

```diff
@@ -46,7 +46,8 @@ export const themes = {
 		messageboxBackground: '#ffffff',
 		searchboxBackground: '#E6E6E7',
 		buttonBackground: '#414852',
-		buttonText: '#ffffff'
+		buttonText: '#ffffff',
+		bannerIcon: '#6d6d72'
```

Comment:
Instead of create a new color, I think you can use `auxiliaryText`, as on other places, or change all places that describe a `cross` icon to use a new color declared here.